neuro_san/internals/journals/tool_argument_reporting.py

```diff
@@ -0,0 +1,59 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+from typing import Any
+from typing import Dict
+from typing import List
+
+from neuro_san.internals.messages.origination import Origination
+
+
+class ToolArgumentReporting:
+    """
+    Utility class to assist in preparing arguments dictionaries when reporing starting a tool.
+    """
+
+    # List of keys for policy objects that cannot be serialized in a message.
+    # These are set in AbstractClassActivation.
+    POLICY_OBJECT_KEYS: List[str] = ["reservationist", "progress_reporter"]
```

Comment:
Make it easier to add more unserializable policy objects to elide from the message reporting.